build/tools/ExtractRuleDescFromJson/RuleDescExtractor.cs

```diff
@@ -98,20 +98,39 @@ private void ProcessRule(Rule rule)
     {
         try
         {
-            // TODO: clean up HTML?
-            SaveRuleFile(rule);
+            var xml = EnsureHtmlIsXml(rule.Description);
+            SaveRuleFile(rule, xml);
         }
         catch (Exception ex)
         {
             Logger.LogError($"Error processing rule. Rule key: {rule.Key}, file: {context.RuleJsonFilePath}, {ex.Message}");
         }
     }
 
-    private void SaveRuleFile(Rule rule)
+    // Regular expression that find empty "col"and "br" HTML elements
+    // e.g. <br>, <br >, <col>, <col span="123">
+    // This is valid HTML, but means we can't parse it as XML. So, we find
+    // the empty elements and replace them with elements with closing tags
+    // e.g. <br>  =>  <br/>
+    // e.g. <col span="123">  =>  <col span="123"/>
+    private static Regex cleanCol = new Regex("(?<element>(<col\\s*)|(col\\s+[^/^>]*))>", RegexOptions.Compiled);
+    private static Regex cleanBr = new Regex("(?<element>(<br\\s*)|(br\\s+[^/^>]*))>", RegexOptions.Compiled);
+
+    private static string EnsureHtmlIsXml(string html)
+    {
+        var xml = html.Replace("&nbsp;", "&#160;");
+
+        xml = cleanCol.Replace(xml, "${element}/>");
+        xml = cleanBr.Replace(xml, "${element}/>");
```

Comment:
The HTML generated for the rule description has a couple of things that stop it from being parseable as XML:

1) it uses entities that are not recognised by the XML reader. Here, the only one we need to worry about is `&nbsp;`, which we just replace with Unicode equivalent.

2) some HTML tags are defined as being "empty" i.e. they don't need a closing tag. The ones we need to worry about are `<col ...>` and `<br>`. We handle this by finding and replacing them by `col .../>` and `<br/>` respectively.